src/test/regress/spec/isolation_shard_rebalancer_progress.spec

```diff
@@ -197,7 +198,8 @@ permutation "s7-grab-lock" "s1-rebalance-c1-block-writes" "s7-get-progress" "s7-
 
 // online rebalancer
 permutation "s6-acquire-advisory-lock" "s1-rebalance-c1-online" "s7-get-progress" "s6-release-advisory-lock" "s1-commit" "s7-get-progress" "enable-deferred-drop"
-permutation "s7-grab-lock" "s1-shard-move-c1-online" "s7-get-progress" "s7-release-lock" "s1-commit" "s7-get-progress" "enable-deferred-drop"
+// Commented out due to flakyness
```

Comment:
Personally I'd like to have some more context on this test is flaky. I see that the commit already has some detailed explanation, so the context is easily reachable, although not in the file content.

Reply:
I'm planning to fix this flakiness later this week. If I cannot for some reason, I'll add some more info to the comment then.